src/modules/events/actions/log-handlers.js

```diff
@@ -40,6 +40,15 @@ export const handleMarketCreatedLog = log => (dispatch, getState) => {
   }
 }
 
+export const handleMarketMigratedLog = log => (dispatch, getState) => {
+  if (log.removed) {
+    dispatch(loadMarketsInfo([log.market]))
+  } else {
+    dispatch(removeMarket(log.market))
```

Comment:
I think instead of only filtering based on the original universe it should filter on if the current universe is _either_ the original or new and then here it should add or remove based on that.

Reply:
ok, will make corrections